README.md

```diff
@@ -183,7 +183,7 @@ Swagger uses the OpenAPI format to specify the API, and this file is found at
 
 ### How to access Swagger
 1. Build and run the server
-2. Access `localhost:5000/swagger` in your browser
+2. Access `localhost:[5000,5050]/swagger` in your browser
```

Comment:
You could split this into two lines, to clarify that port 5000 is for job server docs and 5050 is for auth server docs.